src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -68,6 +62,10 @@ def set_image
         flash[:error] = "KIWI image '#{params[:id]}' does not exist"
         redirect_back(fallback_location: root_path)
       end
+
+      def set_repositories
+        @repositories = @image.repositories.order(:order)
```

Comment:
So you want to write the same code twice because it is important? :laughing: Better would be to write a spec that makes sure that the repositories are displayed in a certain order. Otherwise this is just unnecessary code that we dont need..